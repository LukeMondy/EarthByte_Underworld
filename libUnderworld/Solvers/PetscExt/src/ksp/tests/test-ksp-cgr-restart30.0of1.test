EXEC 		= ./test-ksp-cgr.app -ksp_cgr_restart 30
NPROCS 		= 1
TEST_TYPE       = tolerance_diff_pl,1.0e-10
