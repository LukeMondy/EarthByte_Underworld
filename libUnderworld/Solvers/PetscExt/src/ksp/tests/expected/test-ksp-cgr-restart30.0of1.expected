Init solve 
its 100 
Same rhs 
its 201 
scale rhs solve 
its 291 
perturb rhs solve 
its 470 
Reset search directions 
Init perturbed solve 
its 200 
